src/styles/StyleUtils.js

```diff
@@ -78,11 +78,13 @@ function getNavigationDrawerStyle(isSmallScreenWidth) {
             width: '100%',
             height: '100%',
             borderColor: themeColors.border,
+            backgroundColor: themeColors.appBG,
         }
         : {
             height: '100%',
             width: variables.sideBarWidth,
-            borderRightColor: themeColors.border,
```

Comment:
Okay so I just pulled this down and took it for a spin and it's looking great! That being said, I think we should add this right border back in... it does help create a bit of visual separation between LHN and chat content, and I have a feeling it will be a controversial one if we remove the border.